app/views/mailing_list/steps/completed.html.erb

```diff
@@ -13,24 +13,36 @@
 
     <h2 class="heading-m">What's next?</h2>
 
-    <% if have_degree_low_commitment? %>
-      <%= render(partial: "mailing_list/steps/completed/have_degree_low_commitment") %>
-    <% elsif have_degree_high_commitment? %>
-      <%= render(partial: "mailing_list/steps/completed/have_degree_high_commitment") %>
-    <% elsif final_year_low_commitment? %>
-      <%= render(partial: "mailing_list/steps/completed/final_year_low_commitment") %>
-    <% elsif final_year_high_commitment? %>
-      <%= render(partial: "mailing_list/steps/completed/final_year_high_commitment") %>
-    <% elsif first_or_second_year_low_commitment? %>
-      <%= render(partial: "mailing_list/steps/completed/first_or_second_year_low_commitment") %>
-    <% elsif first_or_second_year_high_commitment? %>
-      <%= render(partial: "mailing_list/steps/completed/first_or_second_year_high_commitment") %>
-    <% elsif no_degree_low_commitment? %>
-      <%= render(partial: "mailing_list/steps/completed/no_degree_low_commitment") %>
-    <% elsif no_degree_high_commitment? %>
-      <%= render(partial: "mailing_list/steps/completed/no_degree_high_commitment") %>
-    <% else %>
-      <%= render(partial: "mailing_list/steps/completed/other_high_low_commitment") %>
+    <% if show_welcome_guide? %>
+      <%= render "mailing_list/steps/completed/shared/welcome_guide" %>
+    <% end %>
+
+    <% if graduate? && high_commitment? %>
+      <%= render "mailing_list/steps/completed/shared/get_an_adviser_high_commitment" %>
+    <% end %>
+
+    <% if (graduate? && low_commitment?) || studying? %>
+      <%= render "mailing_list/steps/completed/shared/get_an_adviser_low_commitment" %>
+    <% end %>
+
+    <% if graduate? || (studying? && low_commitment?) %>
+      <%= render "mailing_list/steps/completed/shared/find_a_teacher_training_course" %>
+    <% end %>
+
+    <% if low_commitment? && (studying? || no_degree?) %>
+      <%= render "mailing_list/steps/completed/shared/is_teaching_right_for_me" %>
+    <% end %>
+
+    <% if no_degree? %>
+      <%= render(partial: "mailing_list/steps/completed/shared/train_to_teach_no_degree") %>
+
+      <p>
+      Consider <a href="/life-as-a-teacher/age-groups-and-specialisms/further-education-teachers">teaching in further education</a> without a degree.
```

Comment:
Not sure why we hardcode this bit here - shouldn't it sit within a partial for consistency @spencerldixon ?

Would suggest this just goes within the `train_to_teach_no_degree` partial and appears for all no degree people to make it easier for now